Source/Csla.Blazor/State/StateManager.cs

```diff
@@ -10,59 +10,100 @@
-    /// </summary>
-    /// <remarks>
-    /// Normally this method is called from the Dispose method
-    /// of a Blazor page, which is the only reliable point
-    /// at which you know the user is navigating to another
-    /// page.
-    /// </remarks>
-    public async Task SaveState(TimeSpan timeout)
-    {
-      var isBrowser = OperatingSystem.IsBrowser();
-      if (isBrowser)
-      {
-         await _sessionManager.SendSession(timeout);
-      }
+        /// <summary>
+        /// Get state from cache.
+        /// </summary>
+        /// <param name="timeout">Time to wait before timing out</param>
+        private async Task GetState(TimeSpan timeout)
+        {
+            var isBrowser = OperatingSystem.IsBrowser();
+            if (isBrowser)
+                _ = await _sessionManager.RetrieveSession(timeout);
+        }
+
+        /// <summary>
+        /// Saves state from Blazor wasm to web server. Must call
+        /// as user navigates to any server-side Blazor page.
+        /// </summary>
+        /// <remarks>
+        /// Normally this method is called from the Dispose method
+        /// of a Blazor page, which is the only reliable point
+        /// at which you know the user is navigating to another
+        /// page.
+        /// </remarks>
+        /// <param name="timeout">Time to wait before timing out</param>
+        public async Task SaveState(TimeSpan timeout)
+        {
+            var isBrowser = OperatingSystem.IsBrowser();
+            if (isBrowser)
+            {
+                await _sessionManager.SendSession(timeout);
+            }
+        }
+
+        /// <summary>
+        /// Get state from cache.
+        /// </summary>
+        /// <param name="ct">Cancellation token</param>
+        public Task InitializeAsync(CancellationToken ct)
+        {
+            return GetState(ct);
+        }
+
+        /// <summary>
+        /// Get state from cache.
+        /// </summary>
+        /// <param name="ct">Cancellation token</param>
+        private async Task GetState(CancellationToken ct)
+        {
+            var isBrowser = OperatingSystem.IsBrowser();
+            if (isBrowser)
+                _ = await _sessionManager.RetrieveSession(ct);
+        }
+
+        /// <summary>
+        /// Saves state from Blazor wasm to web server. Must call
+        /// as user navigates to any server-side Blazor page.
+        /// </summary>
+        /// <remarks>
+        /// Normally this method is called from the Dispose method
+        /// of a Blazor page, which is the only reliable point
+        /// at which you know the user is navigating to another
+        /// page.
+        /// </remarks>
+        /// <param name="ct">Cancellation token</param>
+        public async Task SaveState(CancellationToken ct)
+        {
+            var isBrowser = OperatingSystem.IsBrowser();
+            if (isBrowser)
+            {
+                await _sessionManager.SendSession(ct);
```

Comment:
Even though it's C&P please remove the curly braces for the one liner if

Reply:
In my personal opinion I don't like to remove the curly braces on one lines, i think that in the future we can get any method and add a line.

Anyway, I removed in this case.